### PR TITLE
Fixed an issue with extracting labels from obj literal properties and added support for extracting from object methods

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: packages/babel-plugin/__tests__/styled-macro/__fixtures__/label.js [hunk 1]
```diff
@@ -3,30 +3,4 @@
 import styled from '@emotion/styled/macro'
 import { jsx } from '@emotion/react'
 
-class Thing {
```
Comment (Andarist) [inline packages/babel-plugin/__tests__/styled-macro/__fixtures__/label.js:6]: I've split this file into several, smaller ones. Each new test added to this has been changing generated source maps for each test case (so it has affected unaffected cases) and it was really noisy.

File: packages/babel-plugin/src/utils/label.js [hunk 2]
```diff
@@ -53,6 +53,21 @@ export function getLabelFromPath(path: *, state: *, t: *) {
...
+    (!t.isObjectProperty(path) && !t.isObjectMethod(path)) ||
+    path.node.computed
+  ) {
+    return null
+  }
+  if (t.isIdentifier(path.node.key)) {
+    return path.node.key.name
+  } else {
+    // literals
+    return String(path.node.key.value).replace(/\s+/g, '-')
```
Comment (Andarist) [inline packages/babel-plugin/src/utils/label.js:67]: This handles numeric and bigint literals but I think that I should revert this and only handle string literals here. Thoughts?

Comment (mitchellhamilton) [reply packages/babel-plugin/src/utils/label.js:67]: Yeah, I'd just do string literals

Comment (Andarist) [reply packages/babel-plugin/src/utils/label.js]: done

**Review by mitchellhamilton — APPROVED** [review]

**Pull request merged**